multiqc/modules/supernova/supernova.py

```diff
@@ -113,11 +126,44 @@ def __init__(self):
                 'scale': 'OrRd',
                 'hidden': True
         }
+        self.headers['BC usage'] = {
+                'description': 'fraction of barcodes used; between 0 and 1',
+                'scale': 'OrRd',
+                'hidden': True
```

Comment:
How come so many hidden columns? Is it worth moving these into a separate table instead perhaps?

Reply:
Hmm. I try to make a judgement about what values are the most important to display so as not to clutter this table too much.

There are some loosely defined categories you can put these values in. For example 10X barcode metrics, assembly metrics, values that are computed from the kmer composition.

Reply:
> Hmm. I try to make a judgement about what values are the most important to display so as not to clutter this table too much.

Yup for sure, I'm not suggesting making them all visible in the same table. I'm just thinking about the fact that most people won't realise that these are here at all. So given that there are quite a few, putting them in their own table may make them more accessible.

Does however mean that the scatter plot option won't work against stuff from other modules though.

I leave it up to you! Let me know if you'd like to merge.

Reply:
We still have this [line](https://github.com/ewels/MultiQC/pull/691/files#diff-d6febcd24ea91cf804b5b6d7ff887727L233) in the report. But I'm sure people will miss it :wink:

I can think any tables to split out from the main one that would make sense. So go ahead and merge.

Thanks.